Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/dialog/DialogFragmentTemplateHelper.kt

```diff
@@ -0,0 +1,59 @@
+package de.rki.coronawarnapp.ui.dialog
+
+import androidx.fragment.app.Fragment
+
+typealias DialogAction = () -> Unit
+
+@Suppress("LongParameterList")
+fun Fragment.showDialog(
+    title: String,
+    message: String,
+    positiveButton: String,
```

Comment:
You could use a data class here Button(text, action, color?) to reduce the number of parameters

Reply:
I've tried @mtwalli's approach of creating a DSL. We already had the builder from the `MaterialAlertDialogBuilder` and I've just exposed it and we can now use it's own functions for setting up the dialog. The number of params has now reduced to an acceptable level where `detekt` isn't complaining anymore.